dynamic-config.md

```diff
@@ -54,14 +55,28 @@ show config where name like '%log%'
 show config where type='tikv' and name='log-level'
 ```
 
-### Modify instance configuration
+### Modify TiKV configuration online
 
-To modify configuration according to the instance address and type, use the `set config` SQL statement:
+> **Note:**
+>
+> - After changing TiKV configuration items online, the TiKV configuration file is automatically updated. However, you also need to modify the corresponding configuration items by executing `tiup edit-config`; otherwise, operations like `upgrade` and `reload` will overwrite your changes. For details of modifying configuration items, refer to [Modify configuration using TiUP](/maintain-tidb-using-tiup.md#modify-the-configuration).
```

Comment:
```suggestion
> - After changing TiKV configuration items online, the TiKV configuration file is automatically updated. However, you also need to modify the corresponding configuration items by executing `tiup edit-config`; otherwise, operations such as `upgrade` and `reload` will overwrite your changes. For details of modifying configuration items, refer to [Modify configuration using TiUP](/maintain-tidb-using-tiup.md#modify-the-configuration).
```